components/browser/session/src/main/java/mozilla/components/browser/session/Session.kt

```diff
@@ -174,6 +178,20 @@ class Session(
         !download.consumeBy(consumers)
     }
 
+    /**
+     * Tracker blocking state, true if blocking trackers is enabled, otherwise false.
+     */
+    var trackerBlockingEnabled: Boolean by Delegates.observable(false) { _, old, new ->
+        notifyObservers(old, new) { onTrackerBlockingEnabledChanged(this@Session, trackerBlockingEnabled) }
+    }
+
+    /**
+     * List of URIs that have been blocked in this session.
+     */
+    var trackersBlocked: List<String> by Delegates.observable(emptyList()) { _, old, new ->
+        notifyObservers(old, new) { onTrackerBlocked(this@Session, trackersBlocked.last(), trackersBlocked) }
+    }
+
```

Comment:
So far we only need the `count` in our apps and not the URLs.

Reply:
I saw, yes, but having the list seems like a better solution. It doesn't require additional logic and we can always get the count from the list. It also seemed likely to me that we want to be able to tell *what* was blocked. So, my preference would be to keep the list.

Reply:
Alright. We may be interested in the blocked _categories_ soon. But the GV API doesn't give us that for now anyways.